B/bdwgc/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "bdwgc"
+version = v"8.2.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/ivmai/bdwgc/releases/download/v$(version)/gc-$(version).tar.gz", "f30107bcb062e0920a790ffffa56d9512348546859364c23a14be264b38836a0"),
+    ArchiveSource("https://github.com/ivmai/libatomic_ops/releases/download/v7.6.14/libatomic_ops-7.6.14.tar.gz", "390f244d424714735b7050d056567615b3b8f29008a663c262fb548f1802d292")
```

Comment:
Shouldn't this be a separate package?

Reply:
`bdwgc` requires `libatomic_ops` to be built, see https://github.com/ivmai/bdwgc

![image](https://user-images.githubusercontent.com/13583761/216025275-e82e3f8f-da14-4828-b667-8d23c7725a6c.png)

The documentation does say that we can switch to a compiler that natively supports atomic intrinsics, but the default GCC (4.8.5) cannot meet this requirement. I have not tested other GCC versions, though.

Reply:
Better to use a compiler which supports atomics (perhaps gcc 5 is enough?)